### ISSUE TITLE
Code cleanup

### ISSUE THREAD

**Issue by aurelienb1** [Panopto/Moodle-2.0-plugin-for-Panopto#185]
Hi,

It would be interesting to clean up the code to comply with Moodle standards.

The CodeChecker plugin helps with this task, there is a lot of whitespace, too much space and too long a string.

Another point, what is the point of depending on mod_forum (version.php) since null by in the code there is a call to a class or function of the forum or maybe I have seen wrong.

Regards

**Comment by jrchamp** [comment]
Related: #178

**Comment by jmalmsten-panopto** [comment]
Hi,

I have filed an internal work item to to clean up the block and bring it in line with Moodle standards. I'm not able to directly merge the suggested pull request into our repository, but will take a look at it and can open an internal PR for those changes once we get to this item.

I've also filed an internal item for the mod_forum dependency. Once we confirm that is not needed, I'll make sure it gets removed in a future release.

Thanks,
Joe
